Tema2/smart_objects/SmartCam.java

```diff
@@ -0,0 +1,41 @@
+package smart_objects;
+/*smart security camera
+can be turned on and off
+can start/stop filming*/
+public class SmartCam extends Host{
```

Comment:
SmartCam **is not a** Host.
Host **has a** SmartCam.

Please be attentive of the requirements: it clearly states that the host controls and knows of the the smart home devices. Check the the two OOP relationships questions every time you design a relationship (inheritance or composition)